app/src/main/java/com/sun/funnytoeic/ui/home/HomeActivityViewModel.kt

```diff
@@ -3,22 +3,35 @@ package com.sun.funnytoeic.ui.home
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.viewModelScope
-import com.sun.funnytoeic.data.local.entity.Vocabulary
 import com.sun.funnytoeic.data.repository.VocabularyRepository
 import com.sun.funnytoeic.ui.base.BaseViewModel
+import com.sun.funnytoeic.utils.Constants.VALUE_0
+import com.sun.funnytoeic.utils.Constants.VALUE_1
+import com.sun.funnytoeic.utils.Constants.VALUE_100
 import kotlinx.coroutines.launch
 
 class HomeActivityViewModel(
     private val repository: VocabularyRepository
 ) : BaseViewModel() {
 
-    private val _learnedVocabularies = MutableLiveData<List<Vocabulary>>()
-    val learnedVocabularies: LiveData<List<Vocabulary>>
-        get() = _learnedVocabularies
+    private val _numberVocabularies = MutableLiveData<Int>()
+    private val _numberLearnedVocabularies = MutableLiveData<Int>()
+    private val _currentLevel = MutableLiveData<Int>()
+    val currentLevel: LiveData<Int>
+        get() = _currentLevel
 
     init {
         viewModelScope.launch {
-            _learnedVocabularies.value = repository.getLearnedVocabularies()
+            _numberVocabularies.value = repository.getNumberVocabularies()
+            _numberLearnedVocabularies.value = repository.getNumberLearnedVocabularies()
+            _currentLevel.value = getLearnedPercent()
         }
     }
+
+    private fun getLearnedPercent(): Int {
+        val numberLearned = _numberLearnedVocabularies.value ?: VALUE_0
+        val numberTotal = _numberVocabularies.value ?: VALUE_1
+        return numberLearned * VALUE_100 / numberTotal
```

Comment:
```suggestion
        return numberLearned * MAXIMUM_PROGRESS / numberTotal
```
em k nên đặt tên biến theo giá trị của nó
mà nên đặt tên theo cách nó sử dụng
nếu đặt tên theo giá trị mà em thay đổi giá trị thì e cũng phải thay đổi tên.